test/unit/spec/EnrollCall_spec.js

```diff
@@ -537,6 +557,31 @@ function (Q, Okta, OktaAuth, LoginUtil, Util, Form, Beacon, Expect, $sandbox,
             expect(test.form.errorMessage()).toBe('Your token doesn\'t match our records. Please try again.');
           });
       });
+      itp('triggers an afterError event if error response on verification', function () {
+        return setupAndSendValidCode()
+          .then(function (test) {
+            Q.stopUnhandledRejectionTracking();
+            test.setNextResponse(resActivateError);
+            test.form.setCode(123);
+            test.form.submit();
+            return tick(test);
```

Comment:
can we change all the `ticks` in the tests you are adding for `Expect.waitForFormError`? `tick`s have the potential of adding flakiness.